package.json

```diff
@@ -23,8 +23,7 @@
   "dependencies": {
     "binaryen": "102.0.0-nightly.20211028",
     "long": "^4.0.0",
-    "source-map-support": "^0.5.20",
-    "ts-node": "^6.2.0"
+    "source-map-support": "^0.5.20"
```

Comment:
We used to have `ts-node` here for the case of installing from GH via npm, say `npm i github:AssemblyScript/assemblyscript` or so.

Reply:
The mechanism here is, btw, that the build script removes `dependencies` that are also in `devDependencies` before making a release. See `scripts/prepublish`.